content/300-guides/050-database/100-developing-with-prisma-migrate/180-generating-down-migrations.mdx

```diff
@@ -33,7 +33,7 @@ The `migrate diff` and `db execute` commands are available in Preview in version
 
 When generating a down migration file, there are some considerations to be aware of:
 
-- The down migration can be used to revert your database schema after a failed migration using the steps in [How to apply your down migration to a failed migration](#how-to-apply-your-down-migration-to-a-failed-migration). This requires the use of the `migrate resolve` command, which can only be used on failed migrations. If your up migration was successful and you want to revert it, you will instead need to revert your `prisma.schema` file to its state before the up migration, and generate a new migration with `migrate dev` to .
+- The down migration can be used to revert your database schema after a failed migration using the steps in [How to apply your down migration to a failed migration](#how-to-apply-your-down-migration-to-a-failed-migration). This requires the use of the `migrate resolve` command, which can only be used on failed migrations. If your up migration was successful and you want to revert it, you will instead need to revert your `schema.prisma` file to its state before the up migration, and generate a new migration with `migrate dev` to .
```

Comment:
Unrelated: This looks weird `with `migrate dev` to .` (missing something?)

Reply:
Definitely weird. We could just end it by saying: "If your up migration was successful and you want to revert it, you will need to revert your `schema.prisma` file to its state before the up migration, and then generate a new migration with the `migrate dev` command."

But I suspect there was originally more [and valuable] to the sentence... @keerlu can you take a look?

Reply:
Oops, two brain 🤯s at once...

I think I did literally just want to say what Tana said above, so will add that as a suggestion:

```suggestion
- The down migration can be used to revert your database schema after a failed migration using the steps in [How to apply your down migration to a failed migration](#how-to-apply-your-down-migration-to-a-failed-migration). This requires the use of the `migrate resolve` command, which can only be used on failed migrations. If your up migration was successful and you want to revert it, you will instead need to revert your `schema.prisma` file to its state before the up migration, and generate a new migration with the `migrate dev` command.
```